index.js

```diff
@@ -5,14 +5,35 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
+header.style.color = "blue"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+const profileDiv = document.querySelector("#profile")
+profileDiv.innerHTML = `
+    <img src = ${traveler.photo} alt = ${traveler.name} >
+    <h2> ${traveler.name} </h2>
+    <em> ${traveler.nickname} </em>
+`
+
 
 
 /***** Deliverable 4 *****/
 
+const animalsUl = document.querySelector("#animals")
+animalsUl.innerHTML = 
+    traveler.animalSightings.forEach(function(sighting)) {
```

Comment:
You're on the right track here! But instead of overwriting the HTML, we can create new elements and append them to our `ul` 

. You have several options: 
Change the `innerHTML` of the `animalsUl`
```js
const animalsUl = document.querySelector('ul#animals');
traveler.animalSightings.forEach( function(sighting) {
  animalsUl.innerHTML += `<li data-id="${sighting.id}">
  <p>${sighting.description}</p>
  <img src="${sighting.photo}" alt="${sighting.species}"/>
  <a href="${sighting.link}" target="_blank">Here's a video about the ${sighting.species} species!</a>
  </li>`
});
```
Use the artisanal/innerHTML combo approach
```js
const ul = document.querySelector('ul#animals')
traveler.animalSightings.forEach(function (animalSightingObject) {
  // console.log(animalSightingObject)
  const li = document.createElement('li')
  li.dataset.id = animalSightingObject.id
  li.innerHTML = `<p>${animalSightingObject.description}</p>
  <img src="${animalSightingObject.photo}" alt="${animalSightingObject.species}"/>
  <a href="${animalSightingObject.link}" target="_blank">Here's a video about the ${animalSightingObject.species} species!</a>`
  ul.append(li)
})
```
Using [document fragment](https://developer.mozilla.org/en-US/docs/Web/API/Document/createDocumentFragment) (more advanced)
```js
const fragment = document.createDocumentFragment()
traveler.animalSightings.forEach( function(sightingObject) {
  const li = document.createElement('li')
  li.dataset.id = sightingObject.id
  li.innerHTML = `
  <p>${sightingObject.description}</p>
  <img src="${sightingObject.photo}" alt="${sightingObject.species}"/>
  <a href="${sightingObject.link}" target="_blank">Here's a video about the ${sightingObject.species} species!</a>`
  fragment.append(li)
});
const animalsUl = document.querySelector('ul#animals');
animalsUl.append(fragment)
```